### ISSUE TITLE
[p50] Upgrade from 16.04 to 18.04

### ISSUE THREAD

**Issue by 130s** [kinu-garage/hut_10sqft#464]
Work will soon transition to Ubuntu 18.04, and I also want to use the latest Gazebo 11.0, which is available only on Boinc onward.

**Comment by 130s** [comment]
Keep getting error.
```
$ sudo do-release-upgrade

Calculating the changes

Calculating the changes

Could not calculate the upgrade

An unresolvable problem occurred while calculating the upgrade.

This was likely caused by:
* Unofficial software packages not provided by Ubuntu
Please use the tool 'ppa-purge' from the ppa-purge
package to remove software from a Launchpad PPA and
try the upgrade again.

If none of this applies, then please report this bug using the
command 'ubuntu-bug ubuntu-release-upgrader-core' in a terminal.If
you want to investigate this yourself the log files in
'/var/log/dist-upgrade' will contain details about the upgrade.
Specifically, look at 'main.log' and 'apt.log'.


Restoring original system state

Aborting
Reading package lists... Done
Building dependency tree
Reading state information... Done
=== Command detached from window (Wed Sep 30 08:56:30 2020) ===
=== Command terminated with exit status 1 (Wed Sep 30 08:56:40 2020) ===
```

**Comment by 130s** [comment]
Kept running into an error:
> An unresolvable problem occurred while calculating the upgrade.

Basically https://askubuntu.com/a/1072061/24203 that suggests purging `qt` helped. I did `apt purge libqtcore5* && apt-get autoremove` then the error disappeared.

**Comment by 130s** [comment]
Upgrade is done. Looks like it took a few hours.

Things I noticed so far:
- synergy got uninstalled. Re-installed.
- Can't switch to mozc? I do see mozc_server running. Looks like this is why I switched to MATE on kudu1? https://github.com/130s/hut_10sqft/issues/394#issuecomment-549500646
- Menu shortcut on windows is gone -> Solved by https://github.com/130s/hut_10sqft/issues/394#issuecomment-549500646 ` apt-get install gnome-panel`.

**Comment by 130s** [comment]
> Menu shortcut on windows is gone -> Solved by [#394 (comment)](https://github.com/130s/hut_10sqft/issues/394#issuecomment-549500646) ` apt-get install gnome-panel`.

Looks like it got reverted when OS rebooted (I'm not sure why it rebooted. It happened while I was rearranging power and ethernet cables. Laptop should have battery).
